lib/cli/index.ts

```diff
@@ -85,6 +85,11 @@ const standaloneNodeOption: yargs.Options = {
   describe: 'Start the selenium server standalone with role set to "node".',
   type: 'boolean'
 };
+const GRID_NODE = 'gridNode';
+const gridNodeOption: yargs.Options = {
+  describe: 'Start the selenium grid with role set to "node".',
+  type: 'string'
+};
```

Comment:
I would rebase. STANDALONE is now SELENIUM and SELENIUM_ALIAS

Could we call this as SELENIUM_GRID_URL?
We should use underscores. So the string would be 'selenium_grid_url'

And have one called SELENIUM_GRID_URL_ALIAS?
And this string would be 'standalone_grid_url'

I am trying to standardize this and add a README for this. I have a few flags to fix that I also need to fix.